packages/lib/OidcApi.ts

```
@@ -0,0 +1,242 @@
+import shim from './shim';
+import Logger from '@joplin/utils/Logger';
+import { _ } from './locale';
+import { EventEmitter } from 'events';
+
+const { stringify } = require('query-string');
+const urlUtils = require('./urlUtils.js');
+
+const logger = Logger.create('OidcApi');
+
+export enum OidcEventName {
+	AuthRefreshed = 'authRefreshed',
```

ESLint

[error] 'AuthRefreshed' is defined but never used.

(no-unused-vars)